lzy/allocator/src/main/java/ai/lzy/allocator/alloc/impl/kuber/KuberMountHolderManager.java

```diff
@@ -201,8 +201,7 @@ public static VolumeMount prepareVolumeMount(ServiceConfig.MountConfig mountConf
 
     @NotNull
     public static VolumeRequest createHostPathVolume(ServiceConfig.MountConfig mountConfig) {
-        // TODO: random UUID ?
-        return new VolumeRequest("host-path-volume-" + UUID.randomUUID(),
+        return new VolumeRequest(new RandomIdGenerator().generate("host-path-volume-", 20),
```

Comment:
It's kinda strange to allocate new Generator on each call. Maybe replace it with static method call on generator?

Reply:
it's very simple object
I suppose escape analysis optimization helps us here =)